packages/alfa-dom/src/get-tab-sequence.ts

```diff
@@ -0,0 +1,51 @@
+import { getTabIndex } from "./get-tab-index";
+import { isElement } from "./guards";
+import { traverseNode } from "./traverse-node";
+import { Element, Node } from "./types";
+
+function indexWithin(array: Array<Element>, element: Element) {
+  let lower = 0;
+  let upper = array.length;
+  const index = getTabIndex(element) as number;
+
+  if (index === 0) {
+    return upper;
+  }
+
+  while (lower < upper) {
+    const middle = (lower + (upper - lower) / 2) | 0;
+    const other = getTabIndex(array[middle]) as number;
+    if (other <= index && other !== 0) {
+      lower = middle + 1;
+    } else {
+      upper = middle;
+    }
+  }
+  return lower;
+}
+
+/**
+ * @see https://www.w3.org/TR/html/editing.html#sequential-focus-navigation
+ */
+export function getTabSequence(
+  element: Element,
```

Comment:
The first argument should be a `Node` which will allow any type of node to be used.